zipline/data/minute_bars.py

```diff
@@ -601,32 +610,23 @@ def get_last_traded_dt(self, asset, dt):
 
     def _find_last_traded_position(self, asset, dt):
         volumes = self._open_minute_file('volume', asset)
-        start_date = asset.start_date
+        start_date_minutes = asset.start_date.value / NANOS_IN_MINUTE
+        dt_minutes = dt.value / NANOS_IN_MINUTE
```

Comment:
Not for this PR, but this does make me ask the question, should our `dt` be a `datetime64[m]` when in minute mode, to minimize times that conversions are needed?